Dockerfile

```diff
@@ -26,5 +26,10 @@ RUN poetry config virtualenvs.create false \
     && poetry install --no-root \
     && rm -rf /root/.cache/pypoetry /root/.cache/pip
 
+COPY docker docker
+RUN cd docker/uwsgi-json-logging-plugin \
+    && ./build_plugin.sh \
+    && cd
```

Comment:
```suggestion

```
AFAIK changing directory in `RUN` doesn't affect the next steps as each `RUN` opens a new shell prompt.